### DISCUSSION TITLE
Is it possible to support laravel-orion？

### DISCUSSION THREAD

**Discussion by robin-dongbin** [dedoc/scramble#665] [Ideas]
laravel orion integrates the function of generating openapi documents, but it can only generate endpoints based on it, and the functions are not as rich as this package.

**Comment by prodev443** [comment]
> laravel orion integrates the function of generating openapi documents, but it can only generate endpoints based on it, and the functions are not as rich as this package.

Hello @robin-dongbin 

Have you achieved Scramble + Laravel working together? I'm trying with scramble extensions, but is hard to cover all scenarios.

**Comment by romalytvynenko** [comment]
@robin-dongbin I don't understand how Scramble can support `laravel-orion` 🫣
